src/controllers/productController.ts

```diff
@@ -129,3 +137,258 @@ export async function getProducts(req: any, res: Response) {
   }
   return res.status(200).json(products);
 }
+
+
+export class productController {
```

Comment:
What is the essence of having two classes with the same names having related things in one file? There is this one `productController`